WordOfTheDay.Repository/RepositoryServicesConfiguration.cs

```diff
@@ -15,5 +15,17 @@ public static void AddRepositories(this IServiceCollection services, string conn
 
             services.AddScoped<IWordsRepository, WordsRepository>();
         }
+        public static void AddConfiguredMassTransit(this IServiceCollection services, string host)
```

Comment:
This should not be in repository level. For me it is more for Services level

Reply:
done